File: src/challenges/java/C0201E.java
package challenges.java;

import java.time.LocalDate;
import java.time.temporal.ChronoUnit;

public class C0201E {
	public static void main(String[] args) {
		String date = String.join("-", args);
		
		LocalDate from = LocalDate.now();
		LocalDate to = LocalDate.parse(date);
		
		System.out.printf("There are %d days between %s and %s.", ChronoUnit.DAYS.between(from, to), from, to);
	}
}

File: src/challenges/python/C0201E.py
"""
@author: Michael

http://www.reddit.com/r/dailyprogrammer/comments/2vc5xq/20150209_challenge_201_easy_counting_the_days/
"""
import datetime, sys

a = datetime.datetime.today()
b = datetime.datetime.strptime('-'.join(sys.argv[1:]), '%Y-%m-%d')

print 'There are %d days between %s and %s.' % ((b - a).days, b, a)